src/tests/models/database/sequelize/sequelizeConnector.js

```diff
@@ -178,12 +194,34 @@ function sanitizeDslResult(data) {
     return result;
 }
 
+async function saveFile(id, file) {
+    const fileClient = client.model('file');
+    let params = {
+        id: id,
+        file: file
+    };
+
+    const result = fileClient.create(params);
+    return result;
+}
+
+async function getFile(id) {
+    const fileClient = client.model('file');
+    const options = {
+        attributes: { exclude: ['updated_at', 'created_at'] }
+    };
+    options.where = { id: id };
+    const dbResult = await fileClient.find(options);
+    return dbResult ? dbResult.file : dbResult;
+}
+
 function sanitizeTestResult(data) {
     const result = data.map(function (test) {
         const dataValues = test.dataValues;
         dataValues.artillery_json = JSON.parse(dataValues.artillery_json);
         dataValues.raw_data = JSON.parse(dataValues.raw_data);
         dataValues.id = dataValues.test_id;
+        test.file_id = test.file_id || undefined;
```

Comment:
shouldn't this be dataValues.file_id? If yes and i'm not mistaking, it's weird that tests are not failing (integration test where you create test with file_url and then get the test and check that the file_id exists in the response). Maybe add unit tests in sequelizeConnector: `get test with file_url` since you don't have that test?